### PR TITLE
[WTF-1940] Allow default values for data sources

### PR DESCRIPTION
**Pull request by weirdwater**
This PR adds documentation for the upcoming default values for data sources feature. It is on schedule for 10.16.

It also cleans up some outdated alerts on the page regarding mx9.

### CODE REVIEW

File: content/en/docs/apidocs-mxsdk/apidocs/pluggable-widgets/pluggable-widgets-property-types.md [hunk 1]
```diff
@@ -722,12 +702,29 @@ Only list datasources are supported, therefore specifying `isList="true"` is req
...
+| Attribute      | Required | Attribute Type | Description                                                                                                |
+|----------------|----------|----------------|------------------------------------------------------------------------------------------------------------|
+| `type`         | Yes      | String         | Must be `datasource`                                                                                       |
+| `key`          | Yes      | String         | See [key](#key)                                                                                            |
+| `isList`       | Yes      | Boolean        | Must be `true`                                                                                             |
+| `required`     | No       | Boolean        | This decides if the user is required to specify a datasource, `true` by default                            |
+| `defaultType`  | No       | String         | Default type for the property, supported values are `Database`, `Microflow`, `NanoFlow`, and `Association` |
```
Comment (ancajuravle) [inline content/en/docs/apidocs-mxsdk/apidocs/pluggable-widgets/pluggable-widgets-property-types.md]: No camel case here, so `Nanoflow`.

Comment (weirdwater) [reply content/en/docs/apidocs-mxsdk/apidocs/pluggable-widgets/pluggable-widgets-property-types.md]: Fixed the typo

**Comment by weirdwater** [comment]
Hi @ConnorLand, the PR is now ready for language review.

**Comment by ConnorLand** [comment]
Language looks good @weirdwater, but your PR has some conflicts. I can't see/change them from my end (usually I can but for some reason it isn't showing up in my GH desktop/VSC).

Please resolve your conflicts or use option 2; Close this PR, make a new PR from a fresh fork off dev, and duplicate your changes in a new PR.

**Comment by ConnorLand** [comment]
> Language looks good @weirdwater, but your PR has some conflicts. I can't see/change them from my end (usually I can but for some reason it isn't showing up in my GH desktop/VSC).
> 
> Please resolve your conflicts or use option 2; Close this PR, make a new PR from a fresh fork off dev, and duplicate your changes in a new PR.

This is resolved, thanks!

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**